azurerm/internal/services/appplatform/tests/spring_cloud_service_resource_test.go

```diff
@@ -233,12 +288,17 @@ resource "azurerm_spring_cloud_service" "test" {
     }
   }
 
+  trace {
+    enabled                         = true
+    app_insight_instrumentation_key = azurerm_application_insights.test.instrumentation_key
```

Comment:
we can simplify this down to:

```suggestion
    instrumentation_key = azurerm_application_insights.test.instrumentation_key
```